app/src/main/java/com/pajato/android/gamechat/database/ExperienceManager.java

```diff
@@ -183,11 +183,11 @@ public Experience getExperience(@NonNull final String groupKey, @NonNull final S
             case 0:
             case 1:             // Get the experiences from the rooms in the joined group that have
                                 // experiences and the me room if it has any experiences.
-                String roomKey = expGroupMap.keySet().iterator().next();
+                String groupKey = expGroupMap.keySet().iterator().next(); // this is actually a group key ?!!!??????????????
```

Comment:
Uh, yes it is a group key.  The key set is all the group keys, a collection.  The iterator will sequence through them.  next() obtains the first one, in this case, the only one.